src/pages/all-about-carbon/what-is-carbon.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: What is Carbon?
+description:
+  Carbon is IBM's open source design system for digital products and
+  experiences. With the IBM Design Language as its foundation, the system
+  consists of working code, design tools, and more.
+---
+
+<PageDescription>
+
+Carbon is IBM's open source design system for digital products and experiences.
+With the IBM Design Language as its foundation, the system consists of working
+code, design tools and resources, human interface guidelines, and a vibrant
+community of contributors.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Introduction</AnchorLink>
+<AnchorLink>Guiding principles</AnchorLink>
+<AnchorLink>How we work</AnchorLink>
```

Comment:
There doesn't seem to be a section for this on this page, so this link doesn't work

Reply:
Good catch, @tw15egan !! Thank you. I'll fix now.